docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -1,68 +1,189 @@
+The project MUST have communication channels for users and developers, for example email lists. | (maybe) | Add link to Slack on signalen.org and [email list](https://lists.publiccode.net/mailman/postorius/lists/signalen-discuss.lists.publiccode.net/)?
+The codebase SHOULD have a publicly available roadmap. | no | There is a [product steering group](https://github.com/orgs/Signalen/projects/2), but no roadmap yet.
+The codebase SHOULD advertise the committed engagement of involved organizations in the development and maintenance. | (maybe) | The signalen.org website shows the logos.
+The documentation SHOULD include instructions for how to report potentially security sensitive issues on a closed channel. | no | Document the process in SECURITY.md?
+The codebase SHOULD document the governance of the codebase, contributions and its community, for example in a GOVERNANCE file. | no |
+The codebase MAY include a code of conduct for contributors. |  |
 
-## [Welcome contributors](https://standard.publiccode.net/criteria/open-to-contributions.html)
 
-* Improve [CONTRIBUTING.md](https://github.com/Amsterdam/signals/blob/master/docs/CONTRIBUTING.md), perhaps with content from Amsterdam's [CONTRIBUTING.md](https://github.com/Amsterdam/amsterdam.github.io/blob/master/CONTRIBUTING.md)
 
 ## [Maintain version control](https://standard.publiccode.net/criteria/version-control-and-history.html)
 
-* Good.
-    * Some past commits did not state what they do or why, some just reference an issue, or have "[WIP](https://github.com/Amsterdam/signals/commit/2f9e2f73ee5dc4cdf67d6854d1a7361f6e6aaf9b)" commit messages; developers have become more rigorous.
+- [ ]
+
+Requirement | meets | links and notes
+-----|-----|-----
+You MUST have a way to maintain version control for your code. | yes |
+All files in a codebase MUST be version controlled. | yes |
+All decisions MUST be documented in commit messages. | no | Commit messages often refer to private issues.
+Every commit message MUST link to discussions and issues wherever possible. | partial | Sometimes these are private connections.
+You SHOULD group relevant changes in commits. | yes | Commits are grouped by pull request.
+You SHOULD mark different versions of your codebase, for example using revision tags or textual labels. | yes | Amsterdam tags new releases on Github using Github releases.
+You SHOULD prefer file formats that can easily be version controlled. | yes | Git-friendliness is in the culture.
 
 ## [Require review of contributions](https://standard.publiccode.net/criteria/require-review.html)
 
-* Ensure commits are reviewed including commit message.
+- [ ]
+
+Requirement | meets | links and notes
+-----|-----|-----
+All contributions that are accepted or committed to release versions of the codebase MUST be reviewed by another contributor. | yes |
+Reviews MUST include source, policy, tests and documentation. | | Code yes, but tests and docs/policy not always - maybe a PR template would help?
```

Comment:
Expect some pushback here, anything that increases friction will be looked at carefully.